force-app/main/releaseGating/lwc/releaseManagement/releaseManagement.js

```diff
@@ -51,4 +54,39 @@ export default class EdcReleaseManagement extends NavigationMixin(LightningEleme
                 break;
         }
     }
+
+    handleErrorMessage(event) {
+        event.stopPropagation();
+        this.displayErrorMessage(this.getErrorMessage(event.detail));
+    }
+
+    displayErrorMessage(errorMessage) {
+        console.error(errorMessage);
```

Comment:
It seems this line is throwing an error while running `yarn test:unit --coverage` on my mac.
![image](https://user-images.githubusercontent.com/63684581/131920694-22d23bbc-b727-4be3-8e9e-cb6b7248b247.png)

Reply:
This test is reporting a console error but not failing. The reason for this is to inform an advanced user about an error response from the server in the browser console
```
Test Suites: 11 passed, 11 total
Tests:       53 passed, 53 total
```

Reply:
Now that I know that logging the errors is really important for CCE, I see the need for this. I was just wondering why JEST doesn't like that line. This is not a show stopper, but we should probably look for a workaround or patch to make our test coverage results cleaner.  :-) hehe I originally thought there was something wrong, but I also noticed the test didn't fail.